backend/expense/src/main/java/org/egov/digit/expense/web/validators/BillValidator.java

```diff
@@ -91,11 +91,11 @@ public List<Bill> validateUpdateRequest(BillRequest billRequest) {
 			throw new CustomException("EG_EXPENSE_INVALID_BILL","The bill does not exists for the given combination of "
 					+ " id : " + bill.getId() + " and refernceId : " + bill.getTenantId());
 
-		validateFieldsForUpdate(bill, billsFromSearch.get(0), errorMap);
+//		validateFieldsForUpdate(bill, billsFromSearch.get(0), errorMap);
 
 		Map<String, Map<String, JSONArray>> mdmsData = getMasterDataForValidation(billRequest, bill);
 		validateTenantId(billRequest,mdmsData);
-		validateMasterData(billRequest, errorMap, mdmsData, false);
+//		validateMasterData(billRequest, errorMap, mdmsData, false);
```

Comment:
will this work if deployed in works environment ?

Reply:
made it configurable